### PR TITLE
Shebang, Legend 

### PR DESCRIPTION
**Pull request by SnejPro**
Hi,

- added an shebang to preparedata.py
- improve the styling of the legend

grafik

### CODE REVIEW

File: web/index.html [hunk 1]
```diff
@@ -42,9 +42,14 @@
...
                 padding-left: 20px;
             }
+			
+			.info-col1 button {
+				margin: 5px 0;
+			}
             
             .info-legend {
                 display: none;
+				text-align: left;
```
Comment (M4GNV5) [inline web/index.html]: This looks like mixed indentation again

File: web/index.html [hunk 2]
```diff
@@ -75,16 +80,17 @@
         <div id="coronamap" style="height: 100%; width: 100%;"></div>
 
         <div id="info">
-            <div class="info-col">
+            <div class="info-col info-col1">
                 Stand: <br />
                 <span id="timestamp">...</span><br />
-                <a id="source" href="#">Quelle</a><br />
+                <a id="source" href="#" target="_blank">Quelle</a><br />
+				<button onclick="show_legend()">Zeige Legende</button>
```
Comment (M4GNV5) [inline web/index.html]: mixed indentation

**Review by M4GNV5 — APPROVED** [review]

**Comment by M4GNV5** [comment]
Ups. I pressed approve instead of request changes when submitting the review above.

**Comment by SnejPro** [comment]
Indentation is fixed

**Comment by SnejPro** [comment]
@M4GNV5 Could you push it to the live version? Currently the overlay overlaps very much.

**Comment by M4GNV5** [comment]
done, sorry for the delay

**Pull request merged**